Source/UETorch/Private/UETorch.cpp

```diff
@@ -973,4 +973,65 @@ extern "C" bool DestroyActor(AActor* object) {
 	}
 	object->K2_DestroyActor();
 	return true;
+}
+
+/**
+ * Functions to control pawns
+ */
+
+extern "C" bool SimpleMoveToLocation(UObject* _this, AActor* object, float x, float y, float z) {
```

Comment:
How is this different from `SetLocation()`? And why does it only work for pawns?